introduction.md

```diff
@@ -5,3 +5,12 @@ But I know that the knowledge is improved step-by-step. I'm a person that not ha
 That's enough for today.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?